arithmetization/src/main/java/net/consensys/linea/zktracer/ZkTracer.java

```diff
@@ -54,8 +53,20 @@ public class ZkTracer implements ConflationAwareOperationTracer {
   /** The {@link GasCalculator} used in this version of the arithmetization */
   public static final GasCalculator gasCalculator = new LondonGasCalculator();
 
+  private static final Map<String, Integer> spillings;
+
+  static {
+    try {
+      // Load opcodes configured in src/main/resources/opcodes.yml.
+      OpCodes.load();
+      // Load spillings configured in src/main/resources/spillings.toml.
+      spillings = Utils.computeSpillings();
+    } catch (final Exception e) {
+      throw new RuntimeException(e);
```

Comment:
Is there some typed exception to reuse instead of a plain RuntimeException, maybe it would help debugging if we add the file location.

Reply:
There're no Typed exceptions. The original exception is embedded inside the runtime exception so the user will have all the details related to exception. In this case, the original checked exception is an IOException.

Reply:
I added an error message